### ISSUE TITLE
Work Package Detail Revamp - Scope Tab

### ISSUE THREAD

**Issue by RChandler234** [Northeastern-Electric-Racing/FinishLine#1123]
Blocked by #1121 

Populate the Scope tab with content (should have the same data as the expected activities and deliverables sections did on the old wp detail page, just without the pageblocks, see the project detail page or this PR: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/1112, for an example of how to do that):

This PR should merge into a feature branch called "#1097-wp-detail-revamp" instead of develop.

![image](https://user-images.githubusercontent.com/29521172/234752238-f2aab7d0-9394-4acf-b54b-a732056380be.png)

Epic: #1097
